src/export/StructureDefinitionExporter.ts

```diff
@@ -118,6 +118,9 @@ export class StructureDefinitionExporter implements Fishable {
       if (element) {
         try {
           if (rule instanceof CardRule) {
+            // If only one side of the cardinality is set by the rule, use element's current cardinality
+            if (isNaN(rule.min)) rule.min = element.min;
+            if (rule.max === '') rule.max = element.max;
```

Comment:
Would it be better here to handle these cases inside `element.constrainCardinality` itself? To me that seems a little more self contained. What do you think? If you did make this change you'd also have to add some tests for when min, max, or both are undefined for `ElementDefinition.constrainCardinality`.

Reply:
I like that idea! I was thinking I wanted the true cardinalities that were being constrained be the ones passed into that function, but we do other logical checking within that function, so I can add these to the top of that as well.